File: src/cdva/tests/fixtures/java/multibyte.java
package com.example.greet;

import static org.junit.jupiter.api.Assertions.assertEquals;

import org.junit.jupiter.api.Test;

class Multibyte {
    String greet() {
        return "こんにちは";
    }

    @Test
    void greetsInJapanese() {
        // 挨拶を確かめる
        assertEquals("こんにちは", greet());
    }
}

File: src/cdva/tests/fixtures/kotlin/multibyte.kt
package com.example.greet

import kotlin.test.Test
import kotlin.test.assertEquals

fun greet(): String = "こんにちは"

class GreetingSpec {
    @Test
    fun greetsInJapanese() {
        // 挨拶を確かめる
        assertEquals("こんにちは", greet())
    }
}
